### DISCUSSION TITLE
How do I cite the OrcaSlicer for scientific Papers

### DISCUSSION THREAD

**Discussion by Resos82010** [OrcaSlicer/OrcaSlicer#5867] [Q&A]
I am currently writing on a scientific Paper, in which I used the OrcaSilcer for slicing my 3D-Modells. I have to cite the Orca slicer / have normally to quote the Version and the company who released it. Orca is an open source project on GitHub if I am not mistaking who do I quote to be the "company" behind it?

**Comment by vgdh** [comment]
May be @SoftFever will help you with your question.

**Comment by SoftFever** [comment]
Hi @Resos82010 

Sounds cool!
There is no company behind Orca.
You can quote "SoftFever" if it is required by the paper.
